docs/mochitests.md

```diff
@@ -78,6 +78,19 @@ Any of the below APIs that takes a `url` will match it as a substring, meaning t
 * `waitForThreadEvents(dbg, eventName)` - Waits for specific thread events
 * `waitForDispatch(dbg, type)` - Wait for a specific action type to be dispatch. If an async action, will wait for it to be done.
 
+## Pausing
```

Comment:
You could rename this to `Debugging`, it probably is good to have a small section calling that out, and mentioning the `--jsdebugger` allows you to use the debugger and mention this command. Not sure we need to detail breakpoints/debugger specifically though :)

Thanks for this new function!